.travis.yml

```diff
@@ -148,6 +148,11 @@ jobs:
       env: >-
         FILE_ENV="./ci/test/00_setup_env_native_valgrind.sh"
 
+    - stage: test
+      name: 'x86_64 Linux  [GOAL: install]  [focal]  [no depends, only system libs, c++17]'
```

Comment:
Our releases are built with bionic, and I think there are no plans to drop support for bionic in 0.22.0. So this should probably run on bionic?

Reply:
Done. Travis passes.